src/qibo/backends/__init__.py

```diff
@@ -83,13 +94,17 @@ def __new__(cls):
         return cls._instance
 
     @classmethod
-    def set_backend(cls, backend, platform=None, runcard=None):  # pragma: no cover
+    def set_backend(
+        cls, backend, platform=None, runcard=None, token=None, provider=None
+    ):  # pragma: no cover
```

Comment:
Same proposal of above
```suggestion
    def set_backend(cls, backend, **kwargs):  # pragma: no cover
```